_dev/css/components/checkout.scss

```diff
@@ -440,6 +440,13 @@ body#checkout {
     .forgot-password {
       margin-left: 14.38rem;
     }
+
+    @media (max-width: 425px) {
```

Comment:
I'm so sorry... I missed that while reviewing...

using the `@include media-breakpoint-down(xs)` or `sm` is way better, as we use everywhere else!

Reply:
Yes :) I will fix that